src/include/dawg/root.h

```diff
@@ -56,9 +56,11 @@ class root_model {
 	}
 
 	void do_user_seq(sequence &seq, mutt &m, const subst_model &s, const rate_model &r, residue::data_type b) const {
-		std::copy(root_seq.begin(), root_seq.end(), std::back_inserter(seq));
-		// seq.at(i).rate_cat(r(m));
-		// seq.at(i).branch(b);
+		seq = root_seq;
+		for (auto 묒 = 0; 묒 != seq.size(); ++묒) {
```

Comment:
Why is there a non-latin character here?

Reply:
Thought it might spice up the iteration.

Reply:
And make it harder to maintain code.